src/utils/toastedOperation.ts

```diff
@@ -6,28 +6,57 @@ import TransactionListToast, {
 } from '~/shared/toasts/TransactionListToast'
 import { Layer } from '~/utils/Layer'
 
-export async function toastedOperation(label: string, fn?: () => Promise<void>) {
+export async function toastedOperation(label: string, fn?: () => void | Promise<void>) {
+    await toastedOperations(
+        [
+            {
+                id: uniqueId('operation-'),
+                label: label,
+            },
+        ],
+        () => fn?.(),
+    )
+}
+
+export async function toastedOperations(
```

Comment:
hell yeah! when we merge it, we could also implement it in operator editing cause I had to write handling of possible multiple transactions there manually and it became quite big and complex